docs/decision_record.md

```diff
@@ -6,6 +6,82 @@
 
 [TOC]
 
+## 2022-02-01 - Structure for the return value of the comparison API endpoint
+
+### Decision
+
+The return value MUST return an object following the REQUIRED format specified below.
+
+**REQUIRED**: The endpoint MUST return, in JSON format, an object with these 3 keys: "digests", "arrays", "elements". 
+
+- *digests*: an object with 2 elements, with keys *a* and *b*, and values the level 0 seqcol digests for the compared collections.
```

Comment:
in the case where the sequence collections were posted in full, this implies that the server will have to calculate the sequence collection's digest. 
Since it's not necessary to get the digest to get the comparison, I think this adds unnecessary calculation.
Maybe we should change the property name to `input` or something similar and specify that it is only repeating the input as it was provided.

Reply:
solution: just make the b digest optional.

Reply:
@tcezard I just pushed a change that clarifies this. What do you think?

Reply:
Its much clearer.